mealplanner-ui/src/pages/MealPlans/SearchMeal.tsx

```diff
@@ -124,8 +124,17 @@ export const SearchMeal: React.FC<Props> = ({ data }) => {
         </p>
 
         <Box>
-          {search(searchText).map((m) => (
-            <Button
+          {
```

Comment:
This can be moved to Line 54 where the search is getting filtered, it can be sorted there instead of in the place of rendering.

Reply:
Also another thing is that we have sortBy function in lodash. If I remember correctly lodash is already part of this project. Please check that. If lodash library is already available in our project then you may use that. If not already available, need not add it. Instead you can use your method itself. In any case, I just wanted you to know an alternative way too.

Reply:
I have moved the sort method to the place where the search is getting filtered. Please check and let me know if it looks good.

Reply:
Thanks for providing the alternative solution. But, I will stick with this method itself.